### DISCUSSION TITLE
Project vs project

### DISCUSSION THREAD

**Discussion by pllim** [astropy/astropy-v5.0-paper#96] [Text comments] [answered]
I see a mix of such usage:

* the Astropy Project
* the Astropy project
* the Project
* the project

Is there any difference in meaning when we say "Astropy Project" as opposed to "Astropy project"? If not, we should just pick one and stick with it.

**Comment by adrn** [comment]
I think it should be capital P everywhere?

**Reply by kelle** [reply]
I like capital P everywhere. I also like using the Project as a shorthand.  (I think this language is useful for differentiating the astropy core package from the whole collaboration which includes many packages and non-coding efforts.)

**Comment by adrn** [comment, marked as answer]
I addressed this in a commit that I will push soon. Thanks!
